docker/elasticsearch-setup/Dockerfile

```diff
@@ -1,10 +1,21 @@
 # This "container" is a workaround to pre-create search indices
-FROM jwilder/dockerize:0.6.1
+
+# Defining environment
+ARG APP_ENV=prod
+
+FROM jwilder/dockerize:0.6.1 AS base
 
 RUN apk add --no-cache curl jq
 
-COPY docker/elasticsearch-setup/create-indices.sh /
+FROM base AS prod-install
+
+COPY docker/elasticsearch-setup/create-indices.sh /index
 RUN chmod 755 create-indices.sh
-COPY gms/impl/src/main/resources/index /
+COPY gms/impl/src/main/resources/index /index
```

Comment:
I don't think this prod configuration will work. Please test it out.

- Copies the script to `/index`, which shouldn't exist in this prod configuration, therefore it will literally copy the script to the file `/index`, not `/index/create-indices.sh`
- Run chmod on a file that doesn't exist.
- Copy a directory to a file, which will either fail or override the file.
- The final CMD will try to run a script that doesn't exist.

Reply:
This was a typo while testing out what would make this look better. The script file should still be in root directory. Running both dev and prod flow now to make sure everything works.

Reply:
@jplaisted Running the main docker-compose.yml gets Dockerfiles from docker hub. I mistakenly thought that would test the prod path. Do you know of any good way to test the Dockerfile for the prod path?

Reply:
Try

```
cd docker
./nuke.sh # start from scratch to really verify things
docker compose pull # pull latest images
docker image rm datahub_elasticsearch-setup # delete pulled image
docker-compose build elasticsearch-setup # rebuild image
docker-compose -p datahub up # launch docker without pulling
```

Maybe?

Reply:
Might need to 

```
docker image rm docker image rm docker_elasticsearch-setup
```

Instead. Not clear to me what the image name of the setup image is supposed to be.

Reply:
Awesome! so it's not pulling it from dockerhub but rather just caching it. Thanks for the pointers. 

Everything works correctly!